frontend/static_home_page/css/styles.css

```diff
@@ -1,44 +1,51 @@
 @font-face {
     src: url('../fonts/BCSans-Regular.woff') format("woff");
+    font-weight: 400;
     font-style: normal;
     font-family: 'BCSans';
+    font-display: swap;
 }
 
 @font-face {
     src: url('../fonts/BCSans-BoldItalic.woff') format("woff");
     font-weight: 700;
     font-style: italic;
     font-family: 'BCSans';
+    font-display: swap;
 }
 
 @font-face {
     src: url('../fonts/BCSans-Italic.woff') format("woff");
+    font-weight: 400;
     font-style: italic;
     font-family: 'BCSans';
+    font-display: swap;
 }
 
 @font-face {
     src: url('../fonts/BCSans-Bold.woff') format("woff");
     font-weight: 700;
     font-family: 'BCSans';
+    font-display: swap;
 }
 
 
 :root {
-
     /* Colors: */
-    --focus-blue: #3B99FC;
+    --focus-blue: #0000FF;
```

Comment:
I don't think the new color is the right one. You can refer to our current theme colors here: [Refer](https://github.com/bcgov/onroutebc/blob/main/frontend/src/themes/bcGovStyles.tsx)